hotfixes/wc-api-dev-cheque-defaults.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Prevents the cheque/check payment method from being auto enabled if settings haven't been configured.
+ * It also sets the description/instructions defaults.
+ *
+ * @since 0.8.1
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+function wc_api_dev_cheque_defaults( $settings ) {
+	$settings['enabled']['default'] = 'no';
+	$settings['description']['default'] = __( 'Pay for this order by check.', 'wc-api-dev' );
```

Comment:
@ryelle So that's a good question..

For the other strings in the plugin, the strings are in WooCommerce Core already. These strings are new so they would have to be translated separately. They also won't ever be merged back to WooCommerce core like the API strings.

I *think* we will be breaking these types of changes out of the API plugin very shortly (see the latest thread on our P2: `p90Yrv-1K`).. so those would have a separate text-domain anyway, and a [separate sub-project here](https://translate.wordpress.com/projects/woocommerce/), so they could be translated prior to launching to more users on .com. So in that case, I think it makes sense to use a different text-domain for these changes. I can document that in the README. Does that sound good?